tests/lib.rs

```diff
@@ -251,7 +251,7 @@ fn test_framework_test() {
         .when(TestCommand::ConfirmTest(ConfirmTest {
             test_name: test_name.to_string(),
         }))
-        .then_expect_error_message("test already performed")
+        .then_expect_error_message("test already performed");
```

Comment:
Why remove the semicolons?

Reply:
i didn't, i added a semicolon.
semicolons are optional if the the expression returns the unit value, but i think it's better to use them consistently (and indeed there's a pedantic clippy lint for this)

Reply:
Derp. Thanks!